ambry-store/src/test/java/com/github/ambry/store/CuratedLogIndexState.java

```diff
@@ -1623,11 +1621,10 @@ List<IndexEntry> getValidIndexEntriesForIndexSegment(Offset indexSegmentStartOff
                 if (getExpectedValue(key, true) == null) {
                   if (currentValue.getExpiresAtMs() == Utils.Infinite_Time || currentValue.isTtlUpdate()) {
                     // TODO check validity of permanent delete tombstone
-                    deleteTombstoneStats.get(PERMANENT_DELETE_TOMBSTONE).getFirst().getAndAdd(1);
-                    deleteTombstoneStats.get(PERMANENT_DELETE_TOMBSTONE).getSecond().getAndAdd(currentValue.getSize());
+                    builder.permanentCountInc()
+                        .permanentSizeInc(currentValue.getSize(), currentValue.getOperationTimeInMs());
```

Comment:
Is currentValue good enough? Shall we use finalValue.getOperationTimeInMs to know the delete size?

Reply:
you are right, updated.

Reply:
Thank, approved.